sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/RxGatewayStoreModel.java

```diff
@@ -17,14 +17,14 @@
 import io.netty.buffer.Unpooled;
 import io.netty.handler.codec.http.HttpMethod;
 import io.netty.handler.codec.http.HttpResponseStatus;
+import io.netty.util.internal.StringUtil;
```

Comment:
I would use StringUtils from apache commons lang not from netty

Reply:
This is unused :) 
I will remove it.